Makefile

```diff
@@ -24,7 +24,7 @@ build:
 	go build -o dogechain -ldflags="-X 'github.com/dogechain-lab/dogechain/versioning.Version=$(LATEST_VERSION)+$(COMMIT_HASH)+$(DATE)'" main.go
 
 .PHONY: lint
-lint:
+lint: build
```

Comment:
`lint` target should not rely on `build` target.

Reply:
@DarianShawn  ok, remove it, is my local test command  [e4048e7](https://github.com/dogechain-lab/dogechain/pull/180/commits/e4048e723069beab7022d5e59344f6c3c0153d6c)